sig-installation/kubekey/upgrads.md

```diff
@@ -0,0 +1,39 @@
+# Upgrading Kubernetes and KubeSphere in KubeKey
+## Introduction
+[KubeKey](https://github.com/kubesphere/kubekey) is the new installer of KubeSphere. This document focuses on upgrading Kubernetes and KubeSphere with Kubekey.
+
+Cases of plan support:
+* KubeKey Cluster
+* Non-KubeKey Cluster &ensp; (Cluster deployed with kubeadm)
+
+## KubeKey Cluster
+### Allinone
+Upgrading cluster with a specified version.
+```shell script
+./kk upgrade [--with-kubernetes version] [--with-kubesphere version] 
+```
+* Support upgrading Kubernetes only.
+* Support upgrading KubeSphere only.
+* Support upgrading Kubernetes and KubeSphere.
+
+### Multi-nodes
+Upgrading cluster with a specified configuration file.
+```shell script
+./kk upgrade [(-f | --file) path]
+```
+* Use `-f` to specify the configuration file which was generated for cluster creation.
+* Specifying the target version by parameters in the configuration file.
+
+## Non-KubeKey Cluster
+### Allinone
+Same as KubeKey Cluster.
+### Multi-nodes
+Gathering cluster info and generating kubekey's configuration file.
+```shell script
+./kk gather config [(-f | --file) path] [--kubeconfig path]
```

Comment:
I would say `get` is better than `gather`

Reply:
Yes, I've changed `gather` to `get`.